### DISCUSSION TITLE
v2.228

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#815] [Announcements]
Thank you to everyone for using the project, providing feedback, helping other members, contributing ideas, icons, translations, making patches/features, and more. Special thanks to [Francesco](https://github.com/Francesco146) for maintaining the project during my absence. 

You can join the [Telegram support group chat](https://t.me/AnddeaChat) for discussions, help, and more.

**Comment by icynstant** [comment]
I don't get what YouTube version should I use for v2.228.0, I don't see it in the release description. Should I use v2.227.x versions?

**Reply by anddea** [reply]
Yes, 19.16 is the latest supported, newer versions are less stable for now. 

You can find the supported version in any patching tool or on the main screen of the README for this repository.

19.28 and older are supported except for alternative thumbnails and bypass image restrictions patches.

**Reply by jew4blackberry** [reply]
What are the difference sbetween the two versions of the YouTube mods?

**Reply by anddea** [reply]
No real differences, most of the YouTube features are server side and eventually come to all of them.
